.changes/v3.6.0/816-improvements.md

```diff
@@ -0,0 +1,4 @@
+* `resource/vcd_network_isolated_v2` add support for `metadata` so this field can be set when creating/updating isolated NSX-T networks. [GH-816]
```

Comment:
direct network details missing

Reply:
Done in #804 (for _v1_ networks), as direct networks don't have _v2_

Reply:
ah, ok, only documentation was updated